modules/wolfgame.py

```diff
@@ -5166,6 +5055,7 @@ def transition_night(cli):
         else:
             pm(cli, v_ghost, "You are a \u0002vengeful ghost\u0002.")
         pm(cli, v_ghost, who.capitalize() + ": " + ", ".join(pl))
+        debuglog("GHOST: {0} (Targets: {1}) Players: {2}".format(v_ghost, who.capitalize(), " ".join(pl)))
```

Comment:
1. That's how it is for the message to ghost, but looking at it it does seem pointless. Removed.
2. Well I figured it couldn't hurt to list them, not sure. Modified it so it says something along the lines of: `GHOST: Vgr (target: wolves) - players: nyuszika7h, foo, bar`